mlflow/store/db_migrations/versions/867495a8f9d4_add_trace_tables.py

```diff
@@ -0,0 +1,65 @@
+"""add trace tables
+
+Revision ID: 867495a8f9d4
+Revises: acf3f17fdcc7
+Create Date: 2024-04-27 12:29:25.178685
+
+"""
+from alembic import op
+from mlflow.store.tracking.dbmodels.models import SqlTraceInfo, SqlTraceRequestMetadata, SqlTraceTag
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '867495a8f9d4'
+down_revision = 'acf3f17fdcc7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        SqlTraceInfo.__tablename__,
+        sa.Column("request_id", sa.String(length=50), primary_key=True, nullable=False),
+        sa.Column(
+            "experiment_id",
+            sa.Integer(),
+            sa.ForeignKey("experiments.experiment_id"),
```

Comment:
Can we name the foreign key please? Otherwise MSSQL will generate a generated unique uuid based name that forces additional processing steps for fkey migrations or modifications